PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -351,6 +366,42 @@ To add more Modules, with the understanding that this should be done with cautio
 
 To edit an existing Program, either find the name and click _Edit_ in the options in the right column from the list view or from the _Show_ page, click the EDIT button at the top of the screen. Click on the SAVE button once all the desired information has been changed or entered.
 
+## Name Field Auto-Population
+
+A field can be set to auto-populate based on one or more other fields using the ```auto_populate_list```  in the configuration bundle or system_settings seed file. This is configured to auto-populate the *name* field with the first middle and last names by default.
+
+* ```field_key``` sets the field to be populated.
+
+* ```format``` sets the fields the values are pulled from.
+
+* ```separator``` sets what goes between the pulled values.
+
+* ```auto_populated``` sets if the field_key field is auto-populated.
```

Comment:
Somewhere in here, include a short code snippet example so that the user gets an idea of how to use this feature. Maybe use the "name" from "name_first" + "name_middle" + "name_last" example.